flow-defs/text-field-components.js.flow

```diff
@@ -3,6 +3,9 @@
 import * as React from "react";
```

Comment:
Can we change select arrow asset to mistica icon `chevron-down-regular`? And deprecate the actual one.

![image](https://user-images.githubusercontent.com/6722153/140478432-5bd4cfb1-322b-4dda-aea1-027a9608079b.png)

https://www.figma.com/file/e8d3PkXitUJ1Jwgc9Y3mNc/Forms-Component-Specs?node-id=0%3A1

Reply:
please open a new ticket

Reply:
https://github.com/Telefonica/mistica-web/pull/367